hal/lib/athena/ChipObject.h

```diff
@@ -13,12 +13,6 @@
 #include <stdint.h>
 
 #include "FRC_FPGA_ChipObject/RoboRIO_FRC_ChipObject_Aliases.h"
```

Comment:
I would have made them `NOLINT(build/include_order)`, but there isn't room on the line to fit it. No other checks are disabled by just using `NOLINT`.